README.md

```diff
@@ -1,8 +1,49 @@
-# Fractal integration for Drupal / Twig
-
-This module enables you to build a living style guide based on Fractal within
-Drupal.
-
+# Fractal integration for Twig projects
+
+This module enables you to build a living style guide based on Fractal components using the Twig templating language.
+It can either be used as a Drupal module or as a standalone Twig extension.
+
+## Installation
+Add the following in your project `composer.json` file:
+```json
+"repositories": [
+  {
+    "type": "vcs",
+    "url": "https://github.com/netzstrategen/drupal-twig_fractal"
+  }
+],
+"require": {
+  "drupal/twig_fractal": "dev-master"
```

Comment:
Apparently the GitHub repository name has changed already. But seeing this required package name, shouldn't it be

```suggestion
  "twig/fractal": "dev-master"
```
or
```suggestion
  "netzstrategen/twig-fractal": "dev-master"
```
?

Reply:
Sorry for missing this 🙈😓. IMO we can go with the netzstrategen namespace as the extension is no more strictly bound on Drupal.